src/implementations/vanilla/components/navigation-list/_navigation-list.scss

```diff
@@ -78,6 +78,30 @@ $navigation-list: null !default;
   }
 }
 
+.ecl-navigation-list__item--illustration .ecl-navigation-list__picture {
+  position: absolute;
+  border-radius: 100px 0 100px 100px;
+  top: var(--s-xs);
+  right: var(--s-xs);
+  background: #e0e5f5;
+  height: 60px;
+  width: 60px;
+
+  .ecl-navigation-list__image {
+    border-bottom: none;
+    object-fit: contain;
+    width: auto;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+  }
+
+  + .ecl-navigation-list__content-block .ecl-content-block__title {
+    margin-right: 2.5rem;
```

Comment:
with a long title, the text goes very close to the image. Maybe we could increase the margin here
![image](https://github.com/ec-europa/europa-component-library/assets/25579221/c1c2c441-feab-4d22-af08-d4402fe5d117)

also, on EU the image is getting almost on top of the description
![image](https://github.com/ec-europa/europa-component-library/assets/25579221/e6ea3aa1-15b2-4a20-9782-82fd79fce038)